Sources/Auth/TidalAuth.swift

```diff
@@ -14,14 +14,20 @@ public class TidalAuth: Auth & CredentialsProvider {
 		config: AuthConfig
 	) {
 		self.config = config
+
 		let tokensStore = DefaultTokensStore(credentialsKey: config.credentialsKey, credentialsAccessGroup: config.credentialsAccessGroup)
-		loginRepository = provideLoginRepository(config, tokensStore: tokensStore)
-		tokenRepository = provideTokenRepository(config, tokensStore: tokensStore)
-		
-		AuthLoggable.enableLogging = config.enableLogging
+
+		let authLogger: TidalLogger?
+		if config.enableLogging {
+			authLogger = TidalLogger(label: "Auth", level: .trace)
+		} else {
+			authLogger = nil
+		}
```

Comment:
Q: Why do we want logging to be configurable?

Reply:
From discussions, it seems there's a requirement that the default is nil (which is why the flag is false by default). Another discussion is if we want it to be injected or not, for now it's not injected and therefore not configurable by user or the sdk.